Userland/man.cpp

```diff
@@ -45,43 +46,39 @@ int main(int argc, char* argv[])
 
     unveil(nullptr, nullptr);
 
-    String name;
-    String section;
+    const char* name_str = nullptr;
+    int section = -1;
```

Comment:
No, man sections are actually strings, it just so happens that most of them (currently on Serenity, all of them) look like ints. But there could be sections like `1p` or whatever.